packages/emotion-primitives/src/createEmotion.js

```diff
@@ -0,0 +1,113 @@
+import * as React from 'react'
+import reactPrimitives from 'react-primitives'
+import PropTypes from 'prop-types'
+
+import { getStyles } from './getStyles'
+import { convertToRNStyles } from './convertToRNStyles'
+
+const primitives = ['Text', 'View', 'Image']
+
+const isValidPrimitive = primitive => primitives.indexOf(primitive) > -1
+
+const getPrimitive = primitive => {
+  if (typeof primitive === 'string' && isValidPrimitive(primitive)) {
```

Comment:
Maybe instead of doing this, we could change babel-plugin-emotion to not change the dot syntax if the first character is uppercase?

Reply:
May be we can but I am not sure that `babel-plugin-emotion` will work in other environment. Tweaking this here may work on `web` but not on other platforms.

Reply:
But I really like this idea though 🤔